infrastructure/helm/quantumserverless/README.md

```diff
@@ -59,6 +67,12 @@ Install from specific values file
 kubectl patch svc -n ray kuberay-apiserver-service --type json  --patch '[{"op" : "replace" ,"path" : "/spec/selector" ,"value" : {"app.kubernetes.io/component": "kuberay-apiserver"}}]'
 ```
 
+(temporary) Patch the kuberay-apiserver deployment
+
+```shell
+./hack/apisesrver/patch.sh <LOCAL-IP>
+```
```

Comment:
Similar to my first comment. Probably a patch required by `kuberay-api-server` and not required here?